docs/reference-guide/network-ports.rst

```diff
@@ -0,0 +1,49 @@
+Network Ports
+=============
+
+etcd Ports
+----------
+
+etcd is a cluster of servers storing keys and values
+distributing a database.
+
++----------+-------------------+-------------------+
+|   Port   |  Service/daemon   |     Clients       |
++==========+===================+===================+
+|   2380   |     etcd peer     |    etcd peer      |
++----------+-------------------+-------------------+
+|   2379   |      etcd         |  user, api server |
```

Comment:
I'd put clients in a list rather than comma-separate them.

Also:
- Users don't really interact with the etcd cluster in a K8s environment
- 'API server' or 'kube-apiserver'